docs/user-guide/deploying/release-management.md

```diff
@@ -71,7 +71,7 @@ Another way to migrate an existing project is to manually apply all changes of t
 
 As you can see in the commit the manual migration steps are:
 
-1. Remove the submodule(s)
+1. Remove the directories of the submodule(s)
```

Comment:
This section is about the full migration so it is important the submodule configuration is also removed.

WDYT about:

```suggestion
1. Remove the Git submodule(s) (`.gitmodules`) and each submodule directory (e.g. `openremote`)
```